Presentation.Web/Controllers/API/V2/External/ItSystems/ItSystemV2Controller.cs

```diff
@@ -393,6 +398,51 @@ public IHttpActionResult DeactivateSystemAsRightsHolder([NonEmptyGuid] Guid uuid
                 .Match(_ => StatusCode(HttpStatusCode.NoContent), FromOperationError);
         }
 
+        /// <summary>
+        /// Returns the permissions of the authenticated client in the context of a specific IT-System (a specific IT-System in a specific Organization)
```

Comment:
remove this part `(a specific IT-System in a specific Organization)` as it is incorrect.